addons/packages/load-balancer-and-ingress-service/1.8.1/bundle/config/upstream/loadbalancerandingressservice/rbac.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ako-sa
+  namespace: avi-system
+---
+kind: ClusterRole
```

Comment:
Similarly, need to align with https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/release-1.8.1/helm/ako/templates/clusterrole.yaml